policies/rules.md

```diff
@@ -3,82 +3,88 @@ layout: default
 title: CODEVID-19 Rules
 ---
 
-## Abstract
+## Summary
 
-The CODEVID-19 Hackathon is divided into two phases. Weekly prizes are awarded until April 30th, when judging starts. Following judging final prizes are awarded.
+The CODEVID-19 Hackathon is divided into two phases. Weekly prizes are awarded until April 30th, when projects can submit their projects for the final prizes. Following judging final prizes are awarded.
```

Comment:
The "following judging" grammar thing again.  Gah, GitLab lets me link to other comments, why can't GitHub??

Reply:
You mean [like this](https://github.com/devedmonton/codevid19.com/pull/44#discussion_r399616539) @ajyong?

Just click on the time beside the comment to get a link.